src/main/java/com/minecolonies/coremod/colony/managers/StatisticAchievementManager.java

```diff
@@ -244,7 +244,7 @@ private void incrementStatisticAmount(@NotNull final String statistic)
     public void checkAchievements()
     {
         // the colonies size
-        //final int size = this.citizenManager.getCitizens().size();
+        //final int size = this.citizenManager.getAssignedCitizen().size();
 
         //todo check those later again
         /*if (size >= ModAchievements.ACHIEVEMENT_SIZE_SETTLEMENT)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)